content/post/cheers-to-small-beginnings.md

```diff
@@ -0,0 +1,71 @@
+
+### All aboard
+
+On the initial research we conducted, it was stunning to see everybody excited and speaking out loud about their beliefs and the passion that drove them. “Honest work for fun on funny things” is a glimpse on how we sparkled the team spirit all over this journey.
+
+We knew from the start that albeit design’s written word was still immature, it shouted strong diverse aesthetic: “I don't think a Nile crocodile is serious. It should smile :)” and so the journey from individual to unified vision began.
+
+We believe that design is a team sport. It happens when there is such a knowledge management, transfer and openness within a team that it raises the standards of the critical environment. Design must capacitate everyone involved with the power to surface the value of collective truth, driving the momentum far and near the birth of design’s written word towards its visual translation.
+
+### [Cookin’ on three burners](https://en.wikipedia.org/wiki/Cookin%27_on_3_Burners)
+
+Genuine design happens when we acknowledge that design without rationale is a mere aesthetic exercise. Moreover, the urge for better experiences is hinged on better people not on better design. Bringing it all together, we genuinely praise people above all.
+
+The visual translation started fueling with qualitative data, quantitative data and our company culture. We first started looking for some typography options, and in the process, we stumbled upon [Free Font Ballon](https://www.behance.net/gallery/55482949/Free-Font_Balloon) by the incredibly talented [Aga Magdziak](https://www.behance.net/aga735d).
+
+After a couple of minutes we found out that there was more to Aga than meets the eye. Along with the typeface, Aga created a huge set of visual artifacts that caught our attention. We like to think of what we discovered as open source design, and we deeply believe in it. In the end we came up with a design lexicon - a branch of knowledge - that empowered us to create the Brand Persona, and it was data-oriented all the way.
+
+![{d}ali Lexicon](/post/cheers-to-small-beginnings/dali-lexicon.png)
+
+### Claws that can grab a dinosaur
+
+With a lexicon, we had all we needed to create the Brand Persona’s visual translation. After vivid care on content analysis we chose to give birth to {d}ali, a quite special parrot.
+
+A parrot has the distinct characteristic of easily learning languages, and that pretty much stroked our agenda. Then, we had to make sure that our Brand Persona personality and traits would reflect on {d}ali aesthetics. Its claws translate our brand’s trustworthiness, its tail translate our brand’s enthusiasm and {d}ali’s head is all about being genuine, mainly because it gathers characteristics from several parrots, each one reinforcing our rationale.
```

Comment:
```suggestion
"Why a parrot?" one might ask. First of all, a parrot has the distinct characteristic of easily learning languages: that surely rings us a bell, both in terms of programming languages but also natural languages, from our diverse team. 

Second, had to make sure that our Brand Persona personality and traits were reflected on {d}ali's aesthetics. For example, our brand’s trustworthiness translate into claws, our brand’s enthusiasm into its colorful tail, and {d}ali’s head is all about being genuine, mainly because it gathers characteristics from several parrots, each one reinforcing our rationale.